src/hostname/hostnamectl.c

```diff
@@ -427,11 +427,11 @@ static int show_all_names(sd_bus *bus) {
                             "b",
                             false);
         if (r < 0) {
-                log_full_errno(sd_bus_error_has_names(
+                log_full_errno((sd_bus_error_has_names(
                                                &error,
                                                BUS_ERROR_NO_PRODUCT_UUID,
                                                SD_BUS_ERROR_INTERACTIVE_AUTHORIZATION_REQUIRED,
-                                               SD_BUS_ERROR_UNKNOWN_METHOD) ? LOG_DEBUG : LOG_WARNING,
```

Comment:
As mentioned, it's not necessary to do this for GetProductUUID. The method was introduced along with BUS_ERROR_NO_PRODUCT_UUID.